test/language/expressions/tagged-template/invalid-escape-sequences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Invalid unicode escape sequence in tagged template literals are allowed
+es6id: 11.8.6
```

Comment:
there's no valid `es6id` here, this is after es6 and `esid` is the tag you need to apply with the respective section id in the specs.